Update Cloud_Remote_Access.md (#2559)
SkylineCommunications/dataminer-docs: user-guide/Cloud_Platform/RemoteAccess/Cloud_Remote_Access.md

  To access such files, use the remote access URL followed by `/public/` (e.g. the file *image.png* via `https://ziine-skyline.on.dataminer.services/public/image.png`).

  To make files or webpages available for remote access, add them in the folder `C:\Skyline DataMiner\Webpages\public\` on the DMA.
  
  > [!NOTE]

>   > Files in the folder `C:\Skyline DataMiner\Webpages\public\` will be hosted without authentication on the internal network of that DMA.
>   > Remote access will enforce that only authenticated members of the DMS can access it via dataminer.services.

Reviewer: ```suggestion
  > Files in the folder `C:\Skyline DataMiner\Webpages\public\` will be hosted without authentication on the internal network of a DMA. However, only authenticated members of the DMS will be able to access them via remote access.
```
